autoload/test/base.vim

```diff
@@ -30,8 +30,14 @@ function! test#base#executable(runner) abort
   endif
 endfunction
 
-function! test#base#build_args(runner, args) abort
-  return test#{a:runner}#build_args(a:args)
+function! test#base#build_args(runner, args, strategy) abort
```

Comment:
```suggestion
function! test#base#build_args(runner, args, strategy)
```

Not super familiar with vim script, but do we need to kill the abort if we mean to use exceptions to control the branches of the function?